brave/brave-core
# Update text for `PENDING` status.

Resolves https://github.com/brave/brave-browser/issues/21936.

![rewards_panel](https://user-images.githubusercontent.com/5390451/160900791-6454454f-b66b-4972-b628-ce7e070353b1.png)

![rewards_page](https://user-images.githubusercontent.com/5390451/160900812-fba5ef7d-7bd9-4e49-b2e7-46eed6fb2f83.png)

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. Use Charles Proxy to fake a faulty `https://api.uphold.com/v0/me` endpoint (e.g. `HTTP 500 (Internal Server Error)`).
2. Try verifying with Uphold.
